cfe_internal/update/update_policy.cf

```diff
@@ -247,7 +247,7 @@ bundle agent cfe_internal_update_policy_cpv
       comment => "Copy policy updates from master source on policy server if a new validation was acquired",
       handle => "cfe_internal_update_policy_files_inputs_dir",
       copy_from => u_rcp("$(master_location)", @(update_def.policy_servers)),
-      depth_search => u_recurse("inf"),
+      depth_search => u_infinate_client_policy,
```

Comment:
s/infinate/infinite/ I looked up infinate and it is certainly a word, but looks like a typo.